scripts/test/test_util.js

```diff
@@ -194,35 +192,35 @@ var CommandlineTest = function(command) {
  *   the flag '--resultJsonOutputFile', unless only exitCode is being tested.
  *   For now, this means protractor tests (jasmine/mocha).
  */
-exports.Executor = function() {
-  var tests = [];
-  this.addCommandlineTest = function(command) {
-    var test = new CommandlineTest(command);
-    tests.push(test);
+module.exports = class Executor {
+  constructor() {
+    this.tests = [];
+  }
+
+  addCommandlineTest(command) {
+    const test = new CommandlineTest(command);
+    this.tests.push(test);
     return test;
-  };
+  }
 
-  this.execute = function(logFile) {
-    var failed = false;
+  async execute(logFile) {
+    let failed = false;
 
-    (function runTests(i) {
-      if (i < tests.length) {
-        console.log('running: ' + tests[i].command_);
+    for (let test of this.tests) {
+      try {
+        console.log('running: ' + test.command);
         if (logFile) {
-          tests[i].setTestLogFile(logFile);
+          test.setTestLogFile(logFile);
         }
-        tests[i].run().then(function() {
-          console.log('\n>>> \033[1;32mpass\033[0m');
-        }, function(err) {
-          failed = true;
-          console.log('\n>>> \033[1;31mfail: ' + err.toString() + '\033[0m');
-        }).fin(function() {
-          runTests(i + 1);
-        }).done();
-      } else {
-        console.log('Summary: ' + (failed ? 'fail' : 'pass'));
-        process.exit(failed ? 1 : 0);
+        await test.run();
+        console.log('\n>>> \033[1;32mpass\033[0m');
```

Comment:
```suggestion
        console.log('\n>>> \\033[1;32mpass\\033[0m');
```
Per https://stackoverflow.com/questions/36878850/octal-literals-are-not-allowed-in-strict-mode